blenderproc/python/writer/BopWriterUtility.py

```diff
@@ -16,10 +16,11 @@
 from blenderproc.python.writer.WriterUtility import WriterUtility
 
 
-def write_bop(output_dir: str, target_objects: Optional[List[MeshObject]] = None, depths: Optional[List[np.ndarray]] = None, 
-              colors: Optional[List[np.ndarray]] = None, color_file_format: str = "PNG", dataset: str = "", 
-              append_to_existing_output: bool = True, depth_scale: float = 1.0, jpg_quality: int = 95, save_world2cam: bool = True,
-              ignore_dist_thres: float = 100., m2mm: bool = True, frames_per_chunk: int = 1000):
+def write_bop(output_dir: str, target_objects: Optional[List[MeshObject]] = None, depths: Optional[List[np.ndarray]] = None,
+              colors: Optional[List[np.ndarray]] = None, color_file_format: str = "PNG", dataset: str = "",
+              target_cps: List[str] = ['category_id'], append_to_existing_output: bool = True, depth_scale: float = 1.0,
```

Comment:
@MartinSmeyer and here taret_cps can have an empty list as a default parameter instead